seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/SeatunnelApplication.java

```diff
@@ -25,7 +25,7 @@
 import org.springframework.scheduling.annotation.EnableScheduling;
 import org.springframework.transaction.annotation.EnableTransactionManagement;
 
-@SpringBootApplication
+@SpringBootApplication(scanBasePackages = "org.apache.seatunnel")
```

Comment:
I have a little doubt, (I may not be familiar with SpringBoot), if it is configured as `org.apache.seatunnel`, if this module depends on other modules, will it cause a lot of meaningless scans. (even if it's just a waste of time to scan but has no other effect)

Reply:
If not add this configured , other modules compent can not been scan and loading into container.

Reply:
I mean, can the granularity be a little more fine-grained?

Reply:
Done.